### PR TITLE
TVAULT-5325 lxc packages not installed when using bare metal install

### PR DESCRIPTION
**Pull request by trilio-shiwanand**
This solution will install the lxc packages on Ansible Node.

Before the creation of the lxc-container need to run the install_lxc_packages.yml playbook which resides under 

`../ansible/utils/install_lxc_packages.yml`

### CODE REVIEW

File: ansible/utils/install_lxc_packages.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Install lxc packages
+      package:
+        update_cache: yes
+        name:
+          - "lxc"
+          - "python3-lxc"
+          - "lxc-templates"
+        state: latest
```
Comment (abhijeetpatra) [inline ansible/utils/install_lxc_packages.yml:15]: don't set it to `latest`, we don't want to upgrade the existing ones.
set it to `present` only.

Comment (trilio-shiwanand) [reply ansible/utils/install_lxc_packages.yml]: changed latest to present

File: ansible/utils/install_lxc_packages.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+
+- hosts: localhost
+  become: true
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
```
Comment (abhijeetpatra) [inline ansible/utils/install_lxc_packages.yml:6]: do we need to specify this?
It may vary in the customer's env.

Comment (trilio-shiwanand) [reply ansible/utils/install_lxc_packages.yml:6]: Yes, we need to specify the interpreter.
As per my knowledge, we don't see the issue with it until python3 or a higher version is available on the ansible node.

**Review by abhijeetpatra — APPROVED** [review]

**Pull request merged**